python/pyspark/taskcontext.py

```diff
@@ -140,13 +136,13 @@ class BarrierTaskContext(TaskContext):
     _port = None
     _secret = None
 
-    def __init__(self):
-        """Construct a BarrierTaskContext, use get instead"""
-        pass
-
     @classmethod
     def _getOrCreate(cls):
-        """Internal function to get or create global BarrierTaskContext."""
+        """
+        Internal function to get or create global BarrierTaskContext. We need to make sure
+        BarrierTaskContext returns here because it needs in python worker reuse scenario,
```

Comment:
`We need to make sure
        BarrierTaskContext is returned from here because it is needed in python worker reuse scenario`

Reply:
Thanks! Will address this in next commit.